lib/setExtensions.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * A set of useful extensions to the Set prototype.
+ */
+
+Set.prototype.union = function(other) {
```

Comment:
Monkey patching, ewww :-)

Is there a reason not to just use `_.union()` and `_.map()`?